### PR TITLE
aws-s3api: add page

### PR DESCRIPTION
**Pull request by 258204**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5653

### CODE REVIEW

File: pages/common/aws-s3api.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# aws s3api
+
+> Create and delete s3 buckets, and edit bucket properties.
```
Comment (bl-ue) [inline pages/common/aws-s3api.md:3]:
```suggestion
> Create and delete Amazon S3 buckets, and edit bucket properties.
```

File: pages/common/aws-s3api.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws s3api delete-bucket --bucket {{bucket_name}}`
+
+- List buckets:
+
+`aws s3api list-buckets`
+
+- List the objects inside of a bucket, and only show each object's key and size:
+
+`aws s3api list-objects --bucket {{bucket_name}} --query 'Contents[].{Key: Key, Size: Size}'`
```
Comment (bl-ue) [inline pages/common/aws-s3api.md:20]:
```suggestion
`aws s3api list-objects --bucket {{bucket_name}} --query '{{Contents[].{Key: Key, Size: Size}}}'`
```

File: pages/common/aws-s3api.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# aws s3api
+
+> Create and delete s3 buckets, and edit bucket properties.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/s3api/>.
```
Comment (bl-ue) [inline pages/common/aws-s3api.md:4]: the link says `latest` but it's a liar — the page itself says that it's for v1. Use this link instead:
```suggestion
> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3api/index.html>.
```

File: pages/common/aws-s3api.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws s3api get-object --bucket {{bucket_name}} --key {{object_key}} {{path/to/output_file}}`
+
+- Apply an Amazon S3 bucket policy to a specified bucket:
+
+`aws s3api put-bucket-policy --bucket {{bucket_name}} --policy file://{{path/to/bucket_policy.json}}`
+
+- Download the Amazon S3 bucket policy from a specified bucket:
+
+`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output text > {{path/to/bucket_policy.json}}``
```
Comment (bl-ue) [inline pages/common/aws-s3api.md:36]: Why not add all output types? (from [here](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/index.html#options))
```suggestion
`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output {{json|text|table|yaml|yaml-stream}} > {{path/to/bucket_policy.json}}``
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: pages/common/aws-s3api.md (hunk 4 above)
Comment (bl-ue) [reply pages/common/aws-s3api.md]: Oops, this suggestion got out-of-date.  @258204 do you approve of this change?  If so, I'll re-submit it for you.

Comment (258204) [reply pages/common/aws-s3api.md]: > Oops, this suggestion got out-of-date. @258204 do you approve of this change? If so, I'll re-submit it for you.

Already applied the change after fixing the double ` that CI caught.

**Review by bl-ue — APPROVED** [review]

File: pages/common/aws-s3api.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# aws s3api
+
+> Create and delete Amazon S3 buckets, and edit bucket properties.
```
Comment (mfrw) [inline pages/common/aws-s3api.md:3]: cosmetic nitpick
```suggestion
> Create and delete Amazon S3 buckets and edit bucket properties.
```

File: pages/common/aws-s3api.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Delete a bucket:
+
+`aws s3api delete-bucket --bucket {{bucket_name}}`
+
+- List buckets:
+
+`aws s3api list-buckets`
+
+- List the objects inside of a bucket, and only show each object's key and size:
```
Comment (mfrw) [inline pages/common/aws-s3api.md:18]:
```suggestion
- List the objects inside of a bucket and only show each object's key and size:
```

**Review by mfrw — COMMENTED** [review]
Thank you for your contribution. 
I have few cosmetic nitpicks :)

File: pages/common/aws-s3api.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws s3api put-object --bucket {{bucket_name}} --key {{object_key}} --body {{path/to/file}}`
+
+- Download object from a bucket (The output file is always the last argument):
+
+`aws s3api get-object --bucket {{bucket_name}} --key {{object_key}} {{path/to/output_file}}`
+
+- Apply an Amazon S3 bucket policy to a specified bucket:
+
+`aws s3api put-bucket-policy --bucket {{bucket_name}} --policy file://{{path/to/bucket_policy.json}}`
```
Comment (navarroaxel) [inline pages/common/aws-s3api.md:32]: is `--policy {{smb://path/to/...}}` an alternative?

Comment (bl-ue) [reply pages/common/aws-s3api.md:32]: `smb` and `file` are fundamentally similar, and IMO there's not much need to specify `smb:`.

Comment (258204) [reply pages/common/aws-s3api.md:32]: @bl-ue just out of curiousity, have you managed to get this command to work with smb? Whenever I do it, aws gets confused.

File: pages/common/aws-s3api.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Download the Amazon S3 bucket policy from a specified bucket:
+
+`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output {{json|text|table|yaml|yaml-stream}} > {{path/to/bucket_policy.json}}`
```
Comment (marchersimon) [inline pages/common/aws-s3api.md:36]: If I'd use for example `--output yaml`, I wouldn't get a `.json` file, right? If so, we should probably just say
```suggestion
`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output {{json|text|table|yaml|yaml-stream}} > {{path/to/bucket_policy}}`
```

File: pages/common/aws-s3api.md (hunk 7 above)
Comment (bl-ue) [reply pages/common/aws-s3api.md:32]: Hmm no, I can't get it to work either.
```
An error occurred (MalformedPolicy) when calling the PutBucketPolicy operation: Policies must be valid JSON and the first byte must be '{'
```

Comment (258204) [reply pages/common/aws-s3api.md:32]: In that case we must assume it is not compatible with aws s3api.

Comment (bl-ue) [reply pages/common/aws-s3api.md:32]: I agree — there are likely a very few users of `s3api` + `smb:` + `tldr` 😆

File: pages/common/aws-s3api.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws s3api get-object --bucket {{bucket_name}} --key {{object_key}} {{path/to/output_file}}`
+
+- Apply an Amazon S3 bucket policy to a specified bucket:
+
+`aws s3api put-bucket-policy --bucket {{bucket_name}} --policy file://{{path/to/bucket_policy.json}}`
+
+- Download the Amazon S3 bucket policy from a specified bucket:
+
+`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output {{json|text|table|yaml|yaml-stream}} > {{path/to/bucket_policy}}`
```
Comment (navarroaxel) [inline pages/common/aws-s3api.md:36]: Alphabetic order:
```suggestion
`aws s3api get-bucket-policy --bucket {{bucket_name}} --query Policy --output {{json|table|text|yaml|yaml-stream}} > {{path/to/bucket_policy}}`
```

File: pages/common/aws-s3api.md (hunk 7 above)
Comment (258204) [reply pages/common/aws-s3api.md:32]: If I'm not mistaken, for security reasons, smb should generally be avoided anyways.

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
@marchersimon & @mfrw: can you please approve?

**Review by marchersimon — APPROVED** [review]
👍🏽

**Review by sbrl — APPROVED** [review]
Thanks, @258204!

**Pull request merged**